rln/tests/ffi2.rs

```diff
@@ -0,0 +1,1701 @@
+#[cfg(test)]
+#[cfg(not(feature = "stateless"))]
```

Comment:
Let's make two files with stateless tests and non statless test. This one is quite big